code/API_definitions/qod-api.yaml

```diff
@@ -329,16 +332,32 @@ components:
       type: object
       required:
         - sessionId
-        - event
+        - eventReport
       properties:
         sessionId:
           $ref: "#/components/schemas/SessionId"
-        event:
+        eventReport:
           $ref: "#/components/schemas/SessionEvent"
     SessionEvent:
-      type: string
-      enum:
-        - SESSION_TERMINATED
+      type: object
+      required:
+        - event
+      properties:
+        event:
+          type: string
+          enum:
+            - QOS_STATUS_CHANGED
+        qosStatus:
+          type: object
+          properties:
+            status:
+              $ref: "#/components/schemas/QosStatus"
+            reason:
+              type: string
+              enum:
+                - DURATION_EXPIRES
+                - NETWORK_TERMINATES
+                - UNKNOWN
```

Comment:
I would prefer past tense: DURATION_EXPIRED, NETWORK_TERMINATED

Reply:
I also prefer past tense

Reply:
I also prefer see past participles. But these reason values would be appropriate for a  sort of `session_terminated` event, which I think is necessary explicitly. I don't see them matching the proposed values and definitions for `status`